src/Microsoft.ML.CpuMath/SseIntrinsics.cs

```diff
@@ -19,7 +19,7 @@
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
 using Microsoft.ML.Internal.CpuMath.Core;
-using nuint = System.UInt64;
+using nUInt = System.UInt64;
```

Comment:
Thoughts on name? It didn't like all lowercase.

Reply:
I would prefer lowercase - because this is standard for .net

Reply:
wait. .net have nuint native type. why nuint = uint64??

Reply:
Right - I'm not sure why this is here.  I think the alias should be removed since it's not actually behaving like a native integer.